### ISSUE TITLE
Use of date-tagged ubuntu images?

### ISSUE THREAD

**Issue by mgcrea** [sameersbn/docker-ubuntu#7]
Out of curiosity, why do you use date-tags for your ubuntu images? Wouldn't `ubuntu:trusty` automatically always target the latest date-tag?

```bash
$ docker pull ubuntu
...
$ docker images
REPOSITORY           TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
ubuntu               trusty              d0955f21bf24        4 weeks ago         188.3 MB
ubuntu               latest              d0955f21bf24        4 weeks ago         188.3 MB
```

**Comment by sameersbn** [comment]
@mgcrea The `ubuntu:14.04` tags is sort of a living image. This means that it changes over time. The `ubuntu:14.04` image you pull today will not be the same as the one you pull one month from now. You can think of the `ubuntu:14.04` image as the latest available iteration of `14.04` and not as a tag that represents a distinct point in history when the image was created.

By using the date tagged image I can more of less be sure that the ubuntu image has more or less what I expect. So if you build the image yourself, I can be certain that it will more or less the same image that I built.

As a result this reduces cases where you can have the "it works for me argument" because the starting point in all cases will be the same.

I hope my response makes sense to you.

**Comment by mgcrea** [comment]
@sameersbn Thanks for the detailed response! Indeed it makes sense.
